t/checkout.t

```diff
@@ -5,6 +5,7 @@ use File::Copy 'copy';
 use Mojo::File qw(path tempdir);
 use Mojo::JSON 'decode_json';
 use Cavil::Checkout;
+use Mojo::Util 'dumper';
```

Comment:
Unused import.

Reply:
If it's just for debug use, you can use `warn Mojo::Util::dumper ...;` without import anywhere in code that uses anything with Mojo in the name.